internal/engine/wazevo/backend/isa/arm64/instr.go

```diff
@@ -1342,7 +1343,7 @@ func (a aluOp) String() string {
 	case aluOpUDiv64:
 		return "uDiv64"
 	case aluOpRotR:
-		return "rotR"
+		return "ror"
```

Comment:
is it ok to change the String() representation? Should we also change the constant to `aluOpRoR` ? The reason I changed this was I wanted the listings in the test cases to be valid asm (so that we can double check them in a real assembler)